andela/hc-bigas-opus
# #155965583 Build a FAQ with live chat and help videos

#### What does this PR do?
It integrates the Live chat widget that post the questions to slack while also adding an FAQ model and page 
#### How should this be manually tested?
on the  bottom right there is a green button for chat, click on it and type your question, it should reflect on the slack channel join `https://join.slack.com/t/bigas-opus/signup`the go to `Live Chat` channel.
Also on the on the navigation bar go to the FAQ page and you'll find FAQ's with relevant videos.
<img width="1182" alt="screen shot 2018-04-05 at 19 19 05" src="https://user-images.githubusercontent.com/19773670/38378458-720b7d40-3906-11e8-80ad-4e2f11b59eab.png">
## How the chat looks 
<img width="1198" alt="screen shot 2018-04-05 at 19 21 04" src="https://user-images.githubusercontent.com/19773670/38378489-8ac7e148-3906-11e8-998a-f2e5c54295be.png">
